advanced/configuration.md

```diff
@@ -1,4 +1,69 @@
 # Configuration
 
+Mimic is a fully-configurable app. It allows you to configure a JSON file with the port settings and your projects' details.
 
+The file is called `apimocker.json` and you can find it in the app package content. The config has a JSON structure.
 
+Example Config:
+
+```
+{
+  "httpPort": "3000",
+  "httpsPort": "3001",
+  "projects": [
+    {
+      "uuid": "8f63d210-d2fe-11e8-a311-1b5d7ae5a1b9",
+      "name": "project",
+      "endpoints": [
+        {
+          "uuid": "d9e21a40-d2fe-11e8-a311-1b5d7ae5a1b9",
+          "path": "/show",
+          "method": "GET",
+          "response": {},
+          "request": { "body": {}, "params": "" },
+          "statusCode": 200,
+          "timeout": 0,
+        }
+      ],
+      "fallbackUrlPrefix": { "domain": "", "path": "", "port": "" }
```

Comment:
We should probably just use a simple string like 
`"fallbackUrlPrefix": "https://your.api.host/api/v2/"`